### DISCUSSION TITLE
Sobre Proyecto

### DISCUSSION THREAD

**Discussion by AlexBedoya0803** [rramosp/ai4eng.v1#95] [00 - General]
Hola, profe.

Mi equipo y yo quisiéramos saber si este proyecto es apto para abordarlo como proyecto del curso:

https://www.kaggle.com/c/prediccin-de-estancia-hospitalaria/overview/description

Gracias de antemano.

**Comment by rramosp** [comment]
hola alex, me parece demasiado sencillo ... es una tabla con datos de pacientes para predecir (regresión) la duración de la estadía en el hospital .
